python/railjson_generator/pyproject.toml

```diff
@@ -1,41 +1,12 @@
 [project]
 name = "railjson_generator"
 version = "0.4.9"
-
-[tool.poetry]
-name = "railjson_generator"
-version = "0.4.9"
 description = ""
-authors = ["OSRD <contact@osrd.fr>"]
-
-[tool.poetry.dependencies]
-osrd-schemas = { path = "../osrd_schemas/", develop = false }
-pytest = "^7.4.3"
-pytest-cov = "^4.1.0"
-python = ">=3.9,<3.12"
-
-[tool.poetry.group.dev.dependencies]
-black = "^24.4.2"
-flake8-formatter-junit-xml = "^0.0.6"
-isort = "^5.12.0"
-pyproject-flake8 = "^6.0.0.post1"
-pytype = { platform = "linux", version = "^2023.10.17" }
-
-[tool.flake8]
-ignore = "W503,E203"
-max-line-length = 120
-exclude = [".venv/"]
-
-[tool.black]
-line-length = 120
-
-[tool.isort]
-profile = "black"
-multi_line_output = 3
+authors = [{ name = "OSRD Contributors", email = "contact@osrd.fr" }]
+requires-python = ">= 3.11, < 3.13"
```

Comment:
Can we remove the max version? (for the same reasons as schemas)

Reply:
Done in [7fab4f3](https://github.com/OpenRailAssociation/osrd/compare/1136c8fdcb3034f983b47fef63390a9ec881c0bd..7fab4f32c5b6c6c33700fa1abb0495f754fa9fbb).